integration/presto/src/main/prestosql/org/apache/carbondata/presto/CarbondataModule.java

```diff
@@ -127,7 +127,8 @@ public void configure(Binder binder) {
         .in(Scopes.SINGLETON);
     binder.bind(HivePartitionManager.class).in(Scopes.SINGLETON);
     binder.bind(LocationService.class).to(HiveLocationService.class).in(Scopes.SINGLETON);
-    binder.bind(HiveMetadataFactory.class).in(Scopes.SINGLETON);
+    binder.bind(HiveLocationService.class).to(CarbonDataLocationService.class).in(Scopes.SINGLETON);
```

Comment:
Could you please make one small interaction diagram that contains which all Hive classes carbon extends to support insert into (Similar to what we have for query flow) and upload it in JIRA ?

Reply:
added to jira